docs/UserGuide.md

```diff
@@ -262,7 +271,7 @@ You do not have oyster in your fridge.
 
 ### List expiring foods: `expiring`
 
-Displays a list of food items that are expiring within a week.
+Displays a list of food items that has expired and are expiring within a week.
```

Comment:
should be 'or' instead of 'and'
Displays a list of food items that are expiring within a week or have already expired.